packages/common/src/audius-query/types.ts

```diff
@@ -78,9 +78,17 @@ type EndpointOptions = {
   type?: 'query' | 'mutation'
 }
 
+export type FetchBatchArgs<Args> = Omit<Args, 'id'> & {
+  ids: ID[]
+}
+
 export type EndpointConfig<Args, Data> = {
   fetch: (fetchArgs: Args, context: AudiusQueryContextType) => Promise<Data>
   options: EndpointOptions
+  fetchBatch?: (
+    fetchArgs: { ids: ID[] } & Args,
```

Comment:
```suggestion
    fetchArgs: FetchBatchArgs<Args>,
```

Reply:
Good catch! I was tweaking a bunch of things trying to get the types to work